includes/discovery/functions.inc.php

```diff
@@ -227,7 +227,7 @@ function discover_sensor(&$valid, $class, $device, $oid, $index, $type, $descr,
         $divisor  = 1;
     }
 
-    d_echo("Discover sensor: $oid, $index, $type, $descr, $poller_type, $divisor, $multiplier, $entPhysicalIndex, $current\n");
+    d_echo("Discover sensor: $oid, $index, $type, $descr, $poller_type, $divisor, $multiplier, $entPhysicalIndex, $current, (limits: LL: $low_limit, LW: $low_warn_limit, W: $warn_limit, H: $high_limit)\n");
 
```

Comment:
I did add debugging on limits, this was missing. Please advise if you prefer them out of this PR.

Reply:
No, I've always thought they were missing from the debug output ;)